code/game/objects/items/cigs_lighters.dm

```diff
@@ -270,14 +270,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 
 /obj/item/clothing/mask/cigarette/uplift
 	desc = "An Uplift Smooth brand cigarette."
-	list_reagents = list("nicotine" = 7.5, "menthol" = 7.5)
+	list_reagents = list("nicotine" = 13, "menthol" = 5)

```

Comment:
undocumented?

Reply:
Ah yeah sorry, I forgot to update the CL. It's in my commit titles though:
Menthol cigs taste equally of smoke and mint while still buffing